Toolchain/CMake/LLVMConfig.cmake

```diff
@@ -22,7 +22,13 @@ set(LLVM_INSTALL_UTILS ON CACHE BOOL "")
 set(LLVM_INSTALL_TOOLCHAIN_ONLY OFF CACHE BOOL "Don't install headers, utils, and tools")
 set(LLVM_INSTALL_BINUTILS_SYMLINKS OFF CACHE BOOL "")
 
-set(CLANG_ENABLE_CLANGD OFF CACHE BOOL "")
+if(DEFINED ENV{CLANG_ENABLE_CLANGD} AND "$ENV{CLANG_ENABLE_CLANGD}" STREQUAL "ON")
+    message(STATUS "Enabling clangd and other toolchain build")
```

Comment:
I'm not sure what "and other toolchain build" means here? I would have the text say "enabling clangd as part of toolchain build" or something similar

Reply:
It means other tools like `clang-format`

Reply:
Your message makes more sense to me, I have updated it